src/app/shared/http-services/check-login.service.ts

```diff
@@ -21,11 +21,13 @@ export class CheckLoginService {
     const params = new HttpParams({ fromObject: { login_id: loginId } });
 
     return this.http
-      .get<unknown>(`${appConfig.apiUrl}/current-user/check-login-id`, { params })
+      .get<unknown>(`${appConfig.apiUrl}/current-user/check-login-id`, { params, headers: ++count <= 2 ? { timeout: '50' } : {} })
       .pipe(
         decodeSnakeCase(dataDecoder),
         map(data => data.loginIdMatched),
       );
   }
 
 }
+
+let count = 0;
```

Comment:
Kind of global var? The service are supposed to be created only once if well used. If so, it could be a instance variable instead.

Reply:
This is test content to revert

Reply:
ah sorry